### DISCUSSION TITLE
An Offical (but unsuported) community repository

### DISCUSSION THREAD

**Discussion by jonathan-realman** [pop-os/pop#2078] [Ideas]
Since the newest version comes with its own Repos instead of relying on Ubuntu you would now have the ability to add a community repository(as an opt-in option) so that developers can create packages for PopOS (that can be tested at ones own risk) before they are accepted into the main Repos. This idea is obviously inspired by the AUR. But the AUR, or rather the simplicity it provides downloading lesser known packages with ease, is the main reason people use Arch, followed by others of course.

**Comment by jacobgkau** [comment]
It's worth noting that the AUR just hosts build scripts, it doesn't actually host built packages.

**Reply by jonathan-realman** [reply]
>inspired

**Comment by leviport** [comment]
Have you seen the pop-planet community repo? https://packages.pop-planet.info/

**Reply by jonathan-realman** [reply]
is it actually still usable? It seems borked to me.

**Reply by leviport** [reply]
I thought it was, but I haven't tried it in a while. It may need work, and I'm sure the maintainer would love a hand with it.
